### PR TITLE
Hwp 368 - Create API tests for Community Flags

### PR DESCRIPTION
**Pull request by mattsiel**
# Description

This PR includes the following proposed change(s):

- created a test suite for community flags
- reorganized community tests

## Requires

Add 'Requires Attention' label if appropriate.

- [ ] Requires an update to the .env file
- [ ] Requires adding a file listed in the .gitignore
- [ ] Requires an opinion or answer to a question, see comments.
- [ ] Requires other attention, see below.

If there are any requirements for the developer to make after this PR is merged please list them here.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

For pull requests that are not just a simple fix, please check that the branch works on your local machine.

- [ ] Tested by Zach.
- [ ] Tested by Brandon.
- [ ] Tested by Brady.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [ ] If attention is required by the developer such as updating the .env file, these requirements have been listed.

### CODE REVIEW

File: app/server/tests/communityTests/api-setCommunityFlags.test.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        expect(" " + response.text+ " ").not.toContain(linkFlag);
+    });
+});
+
+describe('Deleting new Community', () => {
+    test('API returns a successful response - code 200', async() => {
+      let response = await community.deleteCommunity(newComTitle, token);
+      expect(response.status).toBe(200);
+    });
+});
```
Comment (dbarkowsky) [inline app/server/tests/communityTests/api-setCommunityFlags.test.js:112]: Needs new line.

**Review by dbarkowsky — CHANGES_REQUESTED** [review]

File: app/server/tests/communityTests/api-deleteCommunityFlags.test.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        expect(" " + response.text+ " ").toContain("Flag not found in query.");
+    });
+});
+
+describe('Deleting new Community', () => {
+    test('API returns a successful response - code 200', async() => {
+      let response = await community.deleteCommunity(newComTitle, token);
+      expect(response.status).toBe(200);
+    });
+});
```
Comment (dbarkowsky) [inline app/server/tests/communityTests/api-deleteCommunityFlags.test.js:99]: Needs new line.

**Review by dbarkowsky — CHANGES_REQUESTED** [review]

File: app/server/tests/communityTests/api-deleteCommunityFlags.test.js (hunk 2 above)
Comment (mattsiel) [inline app/server/tests/communityTests/api-deleteCommunityFlags.test.js:99]:
```suggestion
});

```

File: app/server/tests/communityTests/api-setCommunityFlags.test.js (hunk 1 above)
Comment (mattsiel) [inline app/server/tests/communityTests/api-setCommunityFlags.test.js:112]:
```suggestion
});

```

**Review by dbarkowsky — APPROVED** [review]

**Pull request merged**